### ISSUE TITLE
Preparing for the Addition of Photos

### ISSUE THREAD

**Issue by cmiller02** [risd/wh-info#36]
Hi Jay,

Per Academic Affairs, we'd like to (after Jo takes the photos) have headshots of academic support staff  people on info.  I've heard from a lot of people that they think the only thing info is lacking is headshots, but anyway - for now we'll be starting with the academic support staff (due to their great deal of interaction with students).  Ideally I'd love to have all staff listed on info have their headshot next to them - just since sometimes I email people for months without realizing who they are, and I know that's common around campus.  Just wanted to mention this because Jo will be scheduling a photo shoot day for Acad Supp Staff in the next couple of weeks, I believe, so in case you wanted to start thinking about how we'd place the photos on the site.

Thanks so much!
-Carrie

**Comment by rubillionaire** [comment]
Given that this is a completely new addition to the site, and doesn't fit within any of the existing contexts that the site provides us to add content in, perhaps we can have a conversation about this one with Jay.

**Comment by cmiller02** [comment]
Absolutely, Ruben. Let's continue with other issues in the meantime and this would be great to chat with Jay about. Thanks for this suggestion.
